Notebooks/artist_name_extraction.ipynb

```diff
@@ -0,0 +1,362 @@
+   "id": "bb7ef584",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os\n",
+    "\n",
+    "# Project root\n",
+    "if os.getcwd().endswith(\"Notebooks\"):\n",
+    "    os.chdir(\"..\")\n",
+    "\n",
+    "# Adding the root path to Python path\n",
+    "sys.path.append(os.getcwd())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "742cf1ad",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>image_path</th>\n",
+       "      <th>artist_name</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>raw_data/images/images/Edgar_Degas/Edgar_Degas...</td>\n",
+       "      <td>Edgar Degas</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>raw_data/images/images/Edgar_Degas/Edgar_Degas...</td>\n",
+       "      <td>Edgar Degas</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>raw_data/images/images/Edgar_Degas/Edgar_Degas...</td>\n",
+       "      <td>Edgar Degas</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>raw_data/images/images/Edgar_Degas/Edgar_Degas...</td>\n",
+       "      <td>Edgar Degas</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>raw_data/images/images/Edgar_Degas/Edgar_Degas...</td>\n",
+       "      <td>Edgar Degas</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                          image_path  artist_name\n",
+       "0  raw_data/images/images/Edgar_Degas/Edgar_Degas...  Edgar Degas\n",
+       "1  raw_data/images/images/Edgar_Degas/Edgar_Degas...  Edgar Degas\n",
+       "2  raw_data/images/images/Edgar_Degas/Edgar_Degas...  Edgar Degas\n",
+       "3  raw_data/images/images/Edgar_Degas/Edgar_Degas...  Edgar Degas\n",
+       "4  raw_data/images/images/Edgar_Degas/Edgar_Degas...  Edgar Degas"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from Preprocessing_package.extract_artist_name import create_image_artist_df\n",
```

Comment:
We've updated the files to be lowercase, so this should be `preprocessing_package`